lib/spack/spack/installer.py

```diff
@@ -2197,14 +2180,11 @@ def install(self) -> None:
                 if self.fail_fast:
                     raise InstallError(f"{fail_fast_err}: {str(exc)}", pkg=pkg)
 
-                # Terminate at this point if the single explicit spec has
-                # failed to install.
-                if single_explicit_spec and task.explicit:
-                    raise
-
-                # Track explicit spec id and error to summarize when done
-                if task.explicit:
```

Comment:
There seem to be a change in behavior here. The current behavior seems better (track failure for all build requests), but I have a hard time trying to figure out why before we were using `explicit`.

Reply:
Explicit and build request were seen as the same thing